src/containers/AdminCampaignList.jsx

```diff
@@ -82,17 +82,10 @@ class AdminCampaignList extends React.Component {
     });
   };
 
-  handleChecked = ({ campaignId, checked }) => {
-    this.setState(prevState => {
-      const { campaignsToArchive } = prevState;
-      // checked has to be reversed here because the onTouchTap
-      // event fires before the input is checked.
-      if (!checked) {
-        campaignsToArchive[campaignId] = !checked;
-      } else {
-        delete campaignsToArchive[campaignId];
-      }
-      return { campaignsToArchive };
+  handleChecked = ({ campaignIds }) => {
+    console.log("handleChecked", campaignIds);
```

Comment:
Do we need this `console.log` on prod?

Reply:
Also super excited to be getting this weird `onTouchTap` reversed check behavior out of the code. This was just confusing @jasmosez and I.